src/Parser/Core/Modules/Items/HighfathersMachination.js

```diff
@@ -0,0 +1,54 @@
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import React from 'react';
+import { formatNumber } from 'common/format';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class HighfathersMachination extends Analyzer {
+  static dependencies = {
+      combatants: Combatants,
+  };
+  healing = 0;
+  procUsed = 0;
+  totalProc = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTrinket(ITEMS.HIGHFATHERS_MACHINATION.id);
+  }
+
+  on_byPlayer_heal(event) {
+    const spellId = event.ability.guid;
+
+    if (spellId === SPELLS.AMANTHULS_PRESENCEHEAL.id) {
+      this.procUsed++;
+      this.healing += (event.amount || 0) + (event.absorbed || 0);
+    }
+  }
+
+  on_byPlayer_applybuff(event) {
+      const spellId = event.ability.guid;
+      if (spellId === SPELLS.AMANTHULS_PRESENCEBUFF.id) {
+          this.totalProc++;
+      }
+  }
+
+  on_byPlayer_applybuffstack(event) {
+      const spellId = event.ability.guid;
+      if (spellId === SPELLS.AMANTHULS_PRESENCEBUFF.id) {
+          this.totalProc++;
```

Comment:
As a code style change, can you please update this to use the format: `this.totalProc += 1;`